docs/reference/docs/delete.asciidoc

```diff
@@ -75,7 +74,7 @@ DELETE /twitter/_doc/1?routing=kimchy
 // CONSOLE
 // TEST[continued]
 
-The above will delete a tweet with id 1, but will be routed based on the
+The above will delete a `_doc` with id `1`, but will be routed based on the
```

Comment:
This feels wrong? The documents still represent tweets so it seems correct to continue to speak of deleting tweets (in the abstract) as opposed to the underlying implementation? That is, all mentions of "tweets" should remain as "tweets", all mentions of the type `tweet` should be changed to the type `_doc`.

Reply:
Ok, I will revert these.

Reply:
I did not want to 'implicitly' use the removed type in order to avoid any confusion for user upgrading from 5.x / 6.x.

I will revert these changes